_posts/en/posts/2015-12-23-capacity-increases-faq.md

```diff
@@ -125,6 +133,16 @@ David Harding provided a table of [estimated savings][] at various fee/transacti
 
 Web wallets and exchanges that send large numbers of transactions each day at fixed rates (such as for free or for 1% per spend) are expected to be early adopters---even the small savings per spend seen in the table above will add up to significant amounts of money if repeated hundreds or thousands of times a day.
 
+## I have heard suggestions that this discount is an arbitrary decision to favor payment channels & Lightning over on-chain transactions, is that true? {why-discount}
+
+Witness scripts are discounted because their cost to the network is less than the rest of the transaction data. Unlike outputs they do not go into the UTXO set, and do not need to be maintained in RAM for fast processing of future transactions. They can be pruned from disk as soon as they are validated.
+
+The reason for discounting witnesses is that it makes it easier to spend an output, thereby lowering the dust threshold. It makes it less likely that the UTXO set gets filled up with junk outputs, since small outputs get easier to spend / cleanup.
+
+However, you don't want to make the discount too large because then people will use the witnesses to store their junk on the block chain. Or adversarial miners will fill excess space with random junk to defeat IBLT-like relay schemes.
```

Comment:
s/their junk/general purpose data/

s/chain. Or adversarial miners will/chain, or adversarial miners may/

s/to defeat IBLT-like relay/to defeat fast relay schemes that rely on similar mempools